test/conftest.py

```diff
@@ -160,52 +190,46 @@ def pudl_out_orig(live_dbs, pudl_engine):
     return PudlTabl(pudl_engine=pudl_engine)
 
 
+@pytest.fixture(scope="session")
+def ferc_to_sqlite(live_dbs, pudl_datastore_config, etl_settings):
+    """Create raw FERC 1 SQLite DBs."""
+    if not live_dbs:
+        execute_job(
+            reconstructable(get_ferc_to_sqlite_job),
+            instance=DagsterInstance.get(),
+            run_config={
+                "resources": {
+                    "ferc_to_sqlite_settings": {
+                        "config": etl_settings.ferc_to_sqlite_settings.dict()
+                    },
+                    "datastore": {
+                        "config": pudl_datastore_config,
+                    },
+                },
+            },
+        )
+
+
 @pytest.fixture(scope="session", name="ferc1_engine_dbf")
-def ferc1_dbf_sql_engine(
-    pudl_settings_fixture,
-    live_dbs,
-    ferc_to_sqlite_settings,
-    pudl_datastore_fixture,
-):
+def ferc1_dbf_sql_engine(ferc_to_sqlite):
     """Grab a connection to the FERC Form 1 DB clone.
 
     If we are using the test database, we initialize it from scratch first. If we're
     using the live database, then we just yield a conneciton to it.
     """
-    if not live_dbs:
-        pudl.extract.ferc1.dbf2sqlite(
-            ferc1_to_sqlite_settings=ferc_to_sqlite_settings.ferc1_dbf_to_sqlite_settings,
-            pudl_settings=pudl_settings_fixture,
-            clobber=False,
-            datastore=pudl_datastore_fixture,
-        )
-    engine = sa.create_engine(pudl_settings_fixture["ferc1_db"])
+    engine = sa.create_engine(f"sqlite:///{os.getenv('PUDL_OUTPUT')}/ferc1.sqlite")
     logger.info("FERC1 Engine: %s", engine)
     return engine
 
 
 @pytest.fixture(scope="session", name="ferc1_engine_xbrl")
-def ferc1_xbrl_sql_engine(
-    pudl_settings_fixture,
-    live_dbs,
-    ferc_to_sqlite_settings,
-    pudl_datastore_fixture,
-):
+def ferc1_xbrl_sql_engine(ferc_to_sqlite):
     """Grab a connection to the FERC Form 1 DB clone.
 
     If we are using the test database, we initialize it from scratch first. If we're
     using the live database, then we just yield a conneciton to it.
     """
-    if not live_dbs:
-        pudl.extract.xbrl.xbrl2sqlite(
-            ferc_to_sqlite_settings=ferc_to_sqlite_settings,
-            pudl_settings=pudl_settings_fixture,
-            clobber=False,
-            datastore=pudl_datastore_fixture,
-            workers=5,
-            batch_size=20,
-        )
-    engine = sa.create_engine(pudl_settings_fixture["ferc1_xbrl_db"])
+    engine = sa.create_engine(f"sqlite:///{os.getenv('PUDL_OUTPUT')}/ferc1_xbrl.sqlite")
```

Comment:
Instead of creating the engine we could access it from the `ferc1_xbrl_sqlite_io_manager`:

```python
from pudl.io_managers import ferc1_xbrl_sqlite_io_manager
from dagster import build_init_resource_context

@pytest.fixture(scope="session", name="ferc1_engine_xbrl")
def ferc1_xbrl_sql_engine(ferc_to_sqlite, etl_settings):
       context = build_init_resource_context(resources={"dataset_settings": etl_settings.dataset_settings})
       return ferc1_xbrl_sqlite_io_manager(context).engine
```

Reply:
Much cleaner, fixed